### PR TITLE
Dependency: #927 SSpecies13 SayMods for tongues

### PR DESCRIPTION
**Pull request by DatBoiTim**
## About The Pull Request
Start of SingletonSpecies(SSpecies13) which aims to move any and all behaviors related to species into organs or limbs.

This portion of SSpecies13 moves say_mods entirely onto tongues. Fortunately most of this was in place, had very few tongues I needed to make.
End goal is that species become templates for constructing carbons, and behavior is determined by organs/limbs/etc.

## Why It's Good For The Game
Species Code is Weird. Having them determine carbon behaviors is bad.

## TODO

- [x] Cache tongue on carbon

## Changelog
:cl: DatBoiTim
add: Added species specific tongues where needed
refactor: SayMods are now determined by tongues
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_say.dm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /mob/living/carbon/human/say_mod(input, list/message_mods = list())
-	verb_say = dna.species.say_mod
+	var/obj/item/organ/tongue/T = src.getorganslot(ORGAN_SLOT_TONGUE)
+	verb_say = T.say_mod
```
Comment (Kapu1178) [inline code/modules/mob/living/carbon/human/human_say.dm:3]: Can we cache this to a var on /carbon and have it be set on organ insert?

**Review by Kapu1178 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/human_say.dm (hunk 1 above)
Comment (DatBoiTim) [reply code/modules/mob/living/carbon/human/human_say.dm:3]: Added to to-do

Comment (DatBoiTim) [reply code/modules/mob/living/carbon/human/human_say.dm:3]: That's already a thing.

```getorganslot(organ_slot)``` returns the value of an assoc list at index ```organ_slot```.

The assoc list is already a part of /carbon and is updated whenever an organ is inserted into a specific slot.

Unless you really want me to add a var specifically for their tongue, this is redundant.

Comment (Kapu1178) [reply code/modules/mob/living/carbon/human/human_say.dm:3]:
```/mob/living/carbon/getorgan(typepath)
	return (locate(typepath) in internal_organs)
```
this is stinky

Comment (DatBoiTim) [reply code/modules/mob/living/carbon/human/human_say.dm:3]: That's not what is used.

```
/mob/living/carbon/getorganslot(slot)
	return internal_organs_slot[slot]
```
is how any tongue is being referenced for say_mods.

Comment (Kapu1178) [reply code/modules/mob/living/carbon/human/human_say.dm:3]: ah. Why the fuck do we have so many organ helpers jesus christ

Comment (DatBoiTim) [reply code/modules/mob/living/carbon/human/human_say.dm:3]: > ah. Why the fuck do we have so many organ helpers jesus christ

Something something original coders not knowing about proc overhead or other

Comment (ZephyrTFA) [reply code/modules/mob/living/carbon/human/human_say.dm:3]: If you don't acknowledge the technical debt, it cannot hurt you

**Comment by PrefabQuasar** [comment]
this PR made felenids have the catspeak forced on them without including it anywhere in the changelog or PR

**Comment by PrefabQuasar** [comment]
kinda sus

**Comment by triplezeta** [comment]
bruh

**Comment by Sylphily** [comment]
the fuck did that get through

**Comment by DatBoiTim** [comment]
Unintended.

See #1028

**Pull request merged**